src/AnnotationThread.js

```diff
@@ -364,8 +364,12 @@ class AnnotationThread extends EventEmitter {
      * @return {void}
      */
     deleteSuccessHandler = () => {
-        // Broadcast annotation deletion event
-        this.emit(THREAD_EVENT.delete);
+        if (this.threadID) {
+            this.renderAnnotationPopover();
```

Comment:
Is the logic here that we only destroy the thread if the last annotation in it is deleted? Does the `renderAnnotationPopover` need to be called here or would it already be rendered?